Mage.Sets/src/mage/cards/s/SlicerHiredMuscle.java

```diff
@@ -0,0 +1,123 @@
+import mage.target.targetpointer.FixedTarget;
+import mage.players.Player;
+import mage.abilities.common.SimpleStaticAbility;
+
+/**
+ * @author grimreap124
+ */
+public final class SlicerHiredMuscle extends CardImpl {
+
+    public SlicerHiredMuscle(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.ARTIFACT, CardType.CREATURE }, "{4}{R}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.ROBOT);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+        this.secondSideCardClazz = mage.cards.s.SlicerHighSpeedAntagonist.class;
+
+        // More Than Meets the Eye {2}{R}
+        this.addAbility(new MoreThanMeetsTheEyeAbility(this, "{2}{R}"));
+
+        // Double strike
+        this.addAbility(DoubleStrikeAbility.getInstance());
+
+        // Haste
+        this.addAbility(HasteAbility.getInstance());
+
+        // At the beginning of each opponent's upkeep, you may have that player gain
+        // control of Slicer until end of turn. If you do, untap Slicer, goad it, and it
+        // can't be sacrificed this turn. If you don't, convert it.
+        this.addAbility(new BeginningOfUpkeepTriggeredAbility(Zone.BATTLEFIELD, new SlicerHiredMuscleUpkeepEffect(),
+                TargetController.OPPONENT, false, true));
+
+    }
+
+    private SlicerHiredMuscle(final SlicerHiredMuscle card) {
+        super(card);
+    }
+
+    @Override
+    public SlicerHiredMuscle copy() {
+        return new SlicerHiredMuscle(this);
+    }
+}
+
+class SlicerHiredMuscleUpkeepEffect extends OneShotEffect {
+
+    public SlicerHiredMuscleUpkeepEffect() {
+        super(Outcome.GainControl);
+        staticText = "you may have that player gain control of {this} until end of turn. If you do, untap {this}, goad it, and it can't be sacrificed this turn. If you don't, convert it.";
+    }
+
+    public SlicerHiredMuscleUpkeepEffect(final SlicerHiredMuscleUpkeepEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public SlicerHiredMuscleUpkeepEffect copy() {
+        return new SlicerHiredMuscleUpkeepEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+
+        Permanent sourcePermanent = game.getPermanent(source.getSourceId());
+        Player player = game.getPlayer(sourcePermanent.getOwnerId());
+        Player newController = game.getPlayer(getTargetPointer().getFirst(game, source));
+
+        if (player == null || newController == null || sourcePermanent == null) {
+            return false;
+        }
+
+        if (player.chooseUse(this.getOutcome(), source.getRule(), source, game)) {
+
+            // Gain control
+            ContinuousEffect gainControllEffect = new GainControlTargetEffect(Duration.EndOfTurn, false,
+                    newController.getId());
+            gainControllEffect.setTargetPointer(new FixedTarget(sourcePermanent.getId(), game));
```

Comment:
this can be consistent with the others as `new FixedTarget(sourcePermanent, game)` , since no functional difference:
![image](https://github.com/magefree/mage/assets/12538125/8e00902b-a93a-4016-babd-46ab04156428)

Reply:
Updated to be consistent with the other ability's Slicer gets